packages/pilot/src/containers/EmptyState/AccessDocs/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import InfoCard from '../InfoCard'
+
+const AccessDocs = ({ t }) => (
+  <InfoCard
+    buttonLabel={t('pages.empty_state.access_docs.button_label')}
+    buttonLink={t('pages.empty_state.access_docs.documentation_link')}
+    subtitle={(
+      <span>
+        {t('pages.empty_state.access_docs.subtitle_part_one')}{' '}
```

Comment:
Não rola uma string template aqui ?
```suggestion
        {`${t('pages.empty_state.access_docs.subtitle_part_one')}  `}
```

Reply:
Fica bem melhor, ajustei aqui :heavy_check_mark: